resources/lang/en/document.php

```diff
@@ -13,6 +13,8 @@
     'item' => 'Item',
     'kitchen' => 'kitchen utensils (eg. dishes)',
     'license' => 'Statement',
+    'missing_items' => 'You have not added any items.',
+    'missing_personal_info' => 'Your personal information are missing. Please contanct a system administrator.',
```

Comment:
typo: personal information`s`,  and `contact`

Reply:
Information is uncountable, the correct form is without the 's'.